### ISSUE TITLE
Upload User Data

### ISSUE THREAD

**Issue by brewgator** [cop5725/Movie-Predictor-9000#11]
Currently the only data successfully uploaded in relating to the movie portion of the schema.  This task covers the upload of the user related data tables (`userview`, `userrating`, `location`, `mlensuser`)

**Comment by pt-repos** [comment]
Uploaded User data with demograhics information to the database. I have created the sql files to insert userrating and userview data but i'm not able to insert the data at the moment due to no matching foreign key constraint check. Possibly because not all movies have been uploaded to the database yet.
In total I have taken a subset of 1000 users which gives ~99k rows in each of the ratings and views tables

**Comment by brewgator** [comment]
The movie data should be finished uploading.  Which ID are you using as the `movieID`? It is the tmdbid so if you are using the imdbid that might be why.   We can adjust the references in the schema if it is easier than adjusting the sql files

**Comment by pt-repos** [comment]
I used the movieId in the ratings.csv file which i believe is the tmdbid

**Comment by pt-repos** [comment]
![image](https://user-images.githubusercontent.com/45407561/55087857-1da70880-5081-11e9-9978-8940f9ea7896.png)

**Comment by pt-repos** [comment]
Not all values have a match in the Movie table

**Comment by pt-repos** [comment]
Ok.. I guess you are correct. They have used the imdbid to relate ratings. the movies table has has it as varchar in the form 'ttXXXXXXX'. I'll update the FK in the userrating and format the id to varchar format.

**Comment by pt-repos** [comment]
@ltcarbonell can you add unique constraint in imdbId in Movie table. I'm getting insufficient privileges error.

**Comment by brewgator** [comment]
Yeah I can do that.  We would need to change the reference from tmdbid to imdbid.  I will do that now

**Comment by brewgator** [comment]
Ok it should be good now
